components/utils.py

```diff
@@ -275,6 +279,12 @@ def deploy_blueprint_resource(source, destination, service_name,
                                                   user_resource=user_resource,
                                                   render=render,
                                                   load_ctx=load_ctx)
```

Comment:
Added a comment instead. Extracting to method does not work out so well in this case.